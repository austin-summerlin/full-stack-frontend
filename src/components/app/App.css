body {
  background: rgb(56, 56, 56);
  font-family: 'Lucida Sans Unicode',sans-serif;
}

ul {
  padding: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-wrap: wrap;
  box-sizing: border-box;
  list-style: none;
  text-decoration: none;
}

li{
  text-decoration: none;
  padding-left: 20px;
  padding-right: 20px;
  text-align: center;
  font-size: 9px;
}

a {
  text-decoration: none;
  color: Black;
}

h1{
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}
