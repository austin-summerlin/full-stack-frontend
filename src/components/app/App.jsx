/* eslint-disable max-len */
import React from 'react';
import NewMovie from '../Form/NewMovie';
import MovieList from '../Movie/MovieList';
import Header from './Header';
import Home from '../Home/Home';
import MovieDetails from '../Movie/MovieDetailPage';
import EditMovie from '../Form/EditMovie';
import { Switch, Route } from 'react-router-dom';
import './App.css';


export default function App() {
  return (
    <>
      <Header />
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/movies/" component={MovieList} />
        <Route exact path="/movies/:id" component={MovieDetails} />
        <Route exact path="/movies/add" component={NewMovie} />
        <Route exact path="/movies/:id/edit" component={EditMovie} />
      </Switch>
    </>
  );
}
