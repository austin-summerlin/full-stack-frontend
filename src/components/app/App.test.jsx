import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import App from './App';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<MemoryRouter><App /></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
