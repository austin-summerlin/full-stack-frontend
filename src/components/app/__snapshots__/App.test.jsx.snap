// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <h1>
      Ivy's House of Horror
    </h1>
    <nav>
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        Home
      </a>
      <a
        href="/movies"
      >
        Movies
      </a>
      <a
        href="/movies/add"
      >
        Add Movie
      </a>
    </nav>
  </header>
  <div
    class="Home"
  >
    <header>
      <h2>
        Welcome
      </h2>
    </header>
    <p>
      <a
        href="/movies"
      >
        See the List
      </a>
    </p>
  </div>
</DocumentFragment>
`;
